packages/react/src/ErrorBoundary.spec.tsx

```diff
@@ -167,6 +167,98 @@ describe('<ErrorBoundary/>', () => {
     expect(screen.queryByText(ERROR_MESSAGE)).not.toBeInTheDocument()
     expect(onReset).toHaveBeenCalledTimes(1)
   })
+
+  it('should catch Error by many criteria', async () => {
+    const onErrorParent = vi.fn()
+    const onErrorChild = vi.fn()
+
+    render(
+      <ErrorBoundary fallback={({ error }) => <>{error.message} of Parent</>} onError={onErrorParent}>
+        <ErrorBoundary
+          shouldCatch={[CustomError, (error) => error instanceof CustomError]}
+          fallback={({ error }) => <>{error.message} of Child</>}
+          onError={onErrorChild}
+        >
+          {createElement(() => {
+            throw new Error(ERROR_MESSAGE)
+          })}
+        </ErrorBoundary>
+      </ErrorBoundary>
+    )
+
+    expect(onErrorChild).toBeCalledTimes(0)
+    expect(onErrorParent).toBeCalledTimes(1)
+    await waitFor(() => expect(screen.queryByText(`${ERROR_MESSAGE} of Parent`)).toBeInTheDocument())
+  })
+
+  it('should catch Error by one criteria(ErrorConstructor)', async () => {
```

Comment:
```suggestion
  it('should catch Error by ErrorConstructor', async () => {
```
It seems like a convention, but what about make intentions clear in the TC?!
I think there are many similar types of test codes, the text should be clearer so it will be easier to understand!

(I guess that's because I'm seeing it for the first time. 😅)

Reply:
> It seems like a convention, but what about make intentions clear in the TC?!

@sonsurim It's good for us to make it more clear

> (I guess that's because I'm seeing it for the first time. 😅)

It makes even more sense because you are new to our library. I think it's a better interface if you, a first-time user, can understand it.

Reply:
@manudeli 
> If you okay, Could you make PR to make our code more clear than before?

> Your Pull Request improving these test cases be needed for us too.

Many Thanks! I'll give it a try!